addons/advanced_fatigue/XEH_postInit.sqf

```diff
@@ -1,6 +1,32 @@
 #include "script_component.hpp"
 if (!hasInterface) exitWith {};
 
+#ifdef DEBUG_MODE_FULL
+onEachFrame {
```

Comment:
could this be a separate function?
```#ifdef DEBUG_MODE_FULL
call FUNC(renderDebugLines);
#endif
```

so one could run this without having a debug build?

Reply:
I guess we can do that. But the other debug info (`systemChat` and `hintSilent`) is more helpful.

Should we try to move all of the debug output into a function?